pennylane/devices/default_qutrit.py

```diff
@@ -22,6 +22,8 @@
 import numpy as np
 
 import pennylane as qml  # pylint: disable=unused-import
+import pennylane.math as qnp
```

Comment:
So yes, it does look like we call `pennylane.math` `qnp` in a bunch of places, I think I'd prefer to keep it as `math`.  When I see `qnp`, I think `pennylane.numpy` instead.

This is purely style, and I'm ok with leaving it as `qnp`, but it might be clearer if we left it as `math` or `qmlmath`.